openpype/tools/ayon_sceneinventory/model.py

```diff
@@ -228,52 +231,46 @@ def add_items(self, items, parent=None):
         for item in items:
             grouped[item["representation"]]["items"].append(item)
```

Comment:
Not directly related to this PR but since we are refactoring anyway. Would it make more sense to refactor the word `item` here to `container` since basically what the item **should be** is the `container` data dicts returned from `host.get_containers()` (or legacy `ls()` implementations).

Reply:
Yep, will do.